README.md

```diff
@@ -94,54 +55,62 @@ The SDO owner services are packaged as a single docker container that can be run
 3. Query the ownership vouchers that have already been imported (initially it will be an empty list):
 
    ```bash
+   # either use curl directly
    curl -sS -w "%{http_code}" -u "$HZN_ORG_ID/$HZN_EXCHANGE_USER_AUTH" $HZN_SDO_SVC_URL/vouchers | jq
+   # or use the hzn command, if you have the horizon-cli package installed
+   hzn voucher list
    ```
 
-4. "Ping" the development rendezvous server:
+4. "Ping" the rendezvous server:
 
    ```bash
-   curl -sS -w "%{http_code}" -X POST $SDO_RV_URL/mp/113/msg/20 | jq
+   curl -sS -w "%{http_code}" -H Content-Type:application/json -X POST $SDO_RV_URL/mp/113/msg/20 | jq
    ```
    
 ### <a name="gen-keypair"></a>Generate Owner Key Pairs
 
-For production use of SDO, you need to create 3 key pairs and import them into the owner services container. These key pairs enable you to securely take over ownership of SDO ownership vouchers from SDO-enabled device manufacturers, and to securely configure your booting SDO devices. Use the provided `generate-key-pair.sh` script to easily create the necessary key pairs. (If you are only trying out SDO in a dev/test environment, you can use the sample default key pairs and skip this section. You can always add your own key pairs later.)
+For production use of SDO, you need to create 3 key pairs and import them into the owner services container. These key pairs enable you to securely take over ownership of SDO ownership vouchers from SDO-enabled device manufacturers, and to securely configure your booting SDO devices. Use the provided `generate-key-pair.sh` script to easily create the necessary key pairs. (If you are only trying out SDO in a dev/test environment, you can use the built-in sample key pairs and skip this section. You can always add your own key pairs later.)
 
 1. Go to the directory where you want your generated keys to be saved then download `generate-key-pair.sh`.
 
    ```bash
-   curl -sSLO https://raw.githubusercontent.com/open-horizon/SDO-support/master/keys/generate-key-pair.sh
+   curl -sSLO https://github.com/open-horizon/SDO-support/releases/download/v1.8/generate-key-pair.sh
    chmod +x generate-key-pair.sh
    ```
    
-2. Run `generate-key-pair.sh` script. You will be prompted to answer a few questions in order to produce corresponding certificates to your private keys. You must be using Ubuntu to run this script.
+2. Run the `generate-key-pair.sh` script. You will be prompted to answer a few questions in order to produce certificates for your private keys. (The prompts can be avoided by setting environment variables. See the script source for details.) You must be using Ubuntu to run this script.
```

Comment:
you can type `./generate-key-pair.sh -h` to see the environment variables that need to be set as well